setup1-contributor/src/utils.rs

```diff
@@ -88,16 +88,15 @@ pub fn create_parameters_for_chunk<E: PairingEngine>(
 }
 
 pub fn get_authorization_value<R: Rng + CryptoRng>(
-    private_key: &PrivateKey<Testnet2Parameters>,
+    private_key: &PrivateKey<Testnet2>,
     method: &str,
     path: &str,
     rng: &mut R,
 ) -> Result<String> {
-    let view_key = ViewKey::try_from(private_key)?;
     let address = Address::try_from(private_key)?.to_string();
 
     let message = format!("{} {}", method.to_lowercase(), path.to_lowercase());
-    let signature = hex::encode(&view_key.sign(message.as_bytes(), rng)?.to_bytes_le()?);
+    let signature = hex::encode(&private_key.sign(message.as_bytes(), rng)?.to_bytes_le()?);
```

Comment:
Q: is this change warranted? Not sure if this breaks things (did the address scheme change since the code was written?)

Reply:
cc @howardwu